src/script/calling/callingSubscriptionsHandler.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {QualifiedId} from '@wireapp/api-client/lib/user';
+
+const serializeQualifiedId = ({id, domain}: QualifiedId) => `${id}@${domain}`;
```

Comment:
calling repository has the exact same function. maybe we could extract/merge them to a single location or turn that method to static and use it from there?
https://github.com/wireapp/wire-webapp/blob/dev/src/script/calling/CallingRepository.ts#L742

Reply:
This doesn't need to match the ids that the calling repository is generating. This is internal behavior for the subscription handler and there is no need to leak the implementation between those 2 identities, they can be completely standalone